### PR TITLE
add features and features improvement

### PR DESCRIPTION
**Pull request by mebuki117**
changes:
- add blind evaluation, improve direction/distance
- add misread message
- add direction to stronghold
- add divine feature
- add boat colors (should remove is)
- fixed location to 4, 4
- fixed angle
- if use "use chunk coords" is enable, show is as "Chunk" (if disable, "Location")
- smooth certainty colors

meybe It's not a beautiful code

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,12 +14,12 @@ Config ui:
 - Make sure to enable the API option in the Ninjabrain Bot advanced settings,otherwise the overlay won't work.
 
 ## Getting started
-- Download the [latest release](https://github.com/cylorun/ninbot-overlay/releases/latest).
+- Download the [latest release](https://github.com/mebuki117/ninbot-web-client/releases/latest).
```
Comment (scooterthedev) [inline README.md:17]: gotta fix it back lol

File: README.md [hunk 2]
```diff
@@ -14,12 +14,12 @@ Config ui:
...
 - Run the downloaded file, and a window will open where you can configure stuff.
 - Once set up, open the provided URL in any browser on a device connected to the same local network (PC, phone, laptop, etc.).
 
   ### If you are not on windows
-  - Download the source code, [here](https://github.com/cylorun/ninbot-web-client/archive/refs/heads/main.zip)
+  - Download the source code, [here](https://github.com/mebuki117/ninbot-web-client/archive/refs/heads/more-features.zip)
```
Comment (scooterthedev) [inline README.md:22]: same here

Comment (mebuki117) [inline README.md]: oh. i idiot lol

File: README.md (hunk 1 above)
Comment (scooterthedev) [inline README.md]:
```suggestion
- Download the [latest release](https://github.com/cylorun/ninbot-web-client/releases/latest).
```
gotta fix it back lol

File: README.md (hunk 2 above)
Comment (scooterthedev) [inline README.md]:
```suggestion
  - Download the source code, [here](https://github.com/cylorun/ninbot-web-client/releases)
```
same here

**Comment by scooterthedev** [comment]
PR is erroring out on load, while it works fine with @cylorun's code.
image
Seems like this is the error - 
image

**Pull request merged**